cinema/serializers.py

```diff
@@ -114,9 +114,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
             ValidationError
         )
```

Comment:
It's a code quality issue. You're passing the `ValidationError` class to the `validate_ticket` method. It would be better to raise the `ValidationError` inside the `validate` method itself. This would make your code more readable and maintainable. You can modify the `validate_ticket` method to return a boolean value or an error message, which you can then use in your `validate` method to raise the `ValidationError`.